<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="Port">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PortEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="Type">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>97</y>
      <width>103</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>TCP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>UDP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SSL</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="Save">
    <property name="geometry">
     <rect>
      <x>435</x>
      <y>97</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Send">
    <property name="geometry">
     <rect>
      <x>535</x>
      <y>97</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="SaveTable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>641</width>
      <height>211</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Send</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>To Address</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>To Port</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Type</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ASCII</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Hex</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>0</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/siemensimg/siemens.png);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteSaved">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>130</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Saved Packet</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearContent">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>7</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Contents</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="SendTable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>641</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>To Address</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>To Port</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Method</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ASCII</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Hex</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="clearLog">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Log</string>
    </property>
   </widget>
   <widget class="QLabel" name="Address">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Address</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="AddressEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>100</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="HEX">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HEX</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="HEXEdit">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>70</y>
      <width>561</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ASCII">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ASCII</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ASCIIEdit">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>40</y>
      <width>481</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NameEdit">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>11</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ClearContent</sender>
   <signal>clicked()</signal>
   <receiver>ASCIIEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>570</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ClearContent</sender>
   <signal>clicked()</signal>
   <receiver>AddressEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>570</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ClearContent</sender>
   <signal>clicked()</signal>
   <receiver>NameEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>570</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ClearContent</sender>
   <signal>clicked()</signal>
   <receiver>HEXEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>570</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ClearContent</sender>
   <signal>clicked()</signal>
   <receiver>PortEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>570</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
